package.json

```diff
@@ -61,7 +61,7 @@
     "all": true
   },
   "peerDependencies": {
-    "semantic-release": ">=15.8.0 <18.0.0"
+    "semantic-release": ">=18.0.0"
```

Comment:
If the `peerDependency` of `semantic-release` requires >=18, may need to update the documentation/release notes since installing this version of `changelog` still results unsupported engine warnings.